### PR TITLE
Visit Website Feature

### PR DESCRIPTION
**Pull request by Justus-Becker-Rabbenstein**
- click on 'My Contacts'
- click a single contact
- in the profile is a eye icon to visit the website of the contact
- some design fixes

### CODE REVIEW

File: pages/singleProfile/index.js [hunk 1]
```diff
@@ -85,3 +111,10 @@ const ContainerButtonEdit = styled.button`
   margin-top: 10%;
   margin-bottom: 10%;
 `;
+const ContainerDivFlex = styled.div`
+  display: flex;
+  flex-direction: row;
```
Comment (stefankorder) [inline pages/singleProfile/index.js]: "row" is the standard value of flex-box, so you don't need to set it explicit.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good so far: On line 4 inside singleProfile/index.js you use a template string you don't need, just use the variable. Actually, you don't need the whole object that you're building there. And try to use `===` instead of `==`. Also read my comment.

File: pages/singleProfile/index.js (hunk 1 above)
Comment (Justus-Becker-Rabbenstein) [reply pages/singleProfile/index.js]: - fixed flex: row by deletion
- userArray is an object
- clickedName is a string
- String and object cannot be compared
- the solution is to build an object out of clickedName (name: clickedNameObject) to be able to compare it in the filter() function to the object from UserArray

**Review by actyvystom — COMMENTED** [review]
Good work!

**Pull request merged**